actor/sharedaction/logging.go

```diff
@@ -141,21 +141,23 @@ func GetStreamingLogs(appGUID string, client LogCacheClient) (<-chan LogMessage,
 		const offset = 1 * time.Second
 		walkStartTime := ts.Add(-offset)
 
+		v := func(envelopes []*loggregator_v2.Envelope) bool {
```

Comment:
Also, can you please explain - what you are trying to achieve by moving this outside of the `Walk`?